### DISCUSSION TITLE
Did color theme mode change in the last updates?

### DISCUSSION THREAD

**Discussion by zSynctic** [TomSchimansky/CustomTkinter#1747] [Q&A]
Did color theme mode change or is it not working because when I set it to blue its not even close to blue like the old updates what happened?

**Comment by ghost** [comment]
It's possible the colors of the theme in `blue.json` changed... If that's what you mean? Also the `blue.json` theme looks pretty blue to me :)

**Reply by avalon60** [reply]
Maybe post an image.

**Reply by zSynctic** [reply]
![image](https://github.com/TomSchimansky/CustomTkinter/assets/71632495/593d6090-7cf5-494a-9c75-bcf61f8ab0ec)
the one on the right is an old update using the blue color theme and on the left is the new one

**Reply by ghost** [reply]
So the left image shows the window as a gray without a blue tint. You can just create a new theme or use the old blue theme? Next, the buttons and other widgets are colored blue in the new theme. It's just the window that has been changed.

**Reply by avalon60** [reply]
I can't see anything on those. Doesn't look like an app, just grey, with a couple of light grey squares in the middle?

**Reply by ghost** [reply]
I think its just a tiny screenshot of a CTk window.

**Reply by zSynctic** [reply]
Yeah, anyways thanks

**Comment by ghost** [comment]
On macOS 12.6.6 with CustomTkinter 5.2.0, running `./examples/complex_example.py`, here is the `blue.json` (default) theme:
![Screen Shot 2023-06-21 at 4 51 39 PM](https://github.com/TomSchimansky/CustomTkinter/assets/130259005/6e499de7-f1ee-4d89-87ee-1c4054bbf7f3)

**Reply by avalon60** [reply]
This is from CustomTkinter version 4.6.3:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/89534395/2baf8696-203e-4f08-97ad-99b2c180b162)

I can't say I see much difference.

**Reply by ghost** [reply]
Yep 🤷‍♂️
